src/App.tsx

```diff
@@ -1,26 +1,161 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useRef, useState } from 'react';
+import * as todoManager from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { FILTER, FilterStatus } from './types/Filter';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+import { TodoItem } from './components/TodoItem';
+import { ErrorMessage } from './types/ErrorMassage';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterStatus>(FILTER.ALL);
+  const [deletingIds, setDeletingIds] = useState<Set<number>>(new Set());
```

Comment:
Agree with previous comment - there is no need to use `Set`. Common array is good enough here